app/Repositories/Team/TeamRepository.php

```diff
@@ -90,7 +90,8 @@ public function createTeam($suborganization, $data)
                     $project = $this->projectRepository->find($project_id);
                     if ($project) {
                         // pushed cloning of project in background
-                        CloneProject::dispatch($auth_user, $project, $data['bearerToken'], $suborganization->id, $team)->delay(now()->addSecond());
+                        CloneProject::dispatch($auth_user, $project, $data['bearerToken'], $suborganization->id, $team)
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line